backstop/tests/table.html

```diff
@@ -2378,528 +2453,790 @@ <h2 id="regular">Zebra striping</h2>
-            <div class="iui-tables-cell">$4.75</div>
-          </div>
+            <span class="iui-ellipsis">&mldr;</span>
 
-          <div class="iui-tables-row">
-            <div class="iui-tables-cell">Dog food</div>
-            <div class="iui-tables-cell">$22.95</div>
-          </div>
+            <button
+              class="iui-button iui-borderless"
+              aria-label="Page 30"
+              type="button"
+            >
+              <span class="iui-label">30</span>
+            </button>
+          </span>
 
-          <div class="iui-tables-row">
-            <div class="iui-tables-cell">Fish</div>
-            <div class="iui-tables-cell">$16.54</div>
-          </div>
+          <button
+            class="iui-button iui-borderless"
+            aria-label="Next page"
+            type="button"
+          >
+            <!-- Use chevron-right.svg from @itwin/itwinui-icons package. -->
+            <svg
+              class="iui-icon"
+              aria-hidden="true"
+              viewBox="0 0 16 16"
+            >
+              <path d="m4.7 0-1.4 1.4 6.6 6.6-6.6 6.6 1.4 1.4 8-8z" />
+            </svg>
+          </button>
         </div>
 
-        <div class="iui-tables-row iui-tables-paginator">
-          <div class="iui-content">
-            <div class="iui-full-line"></div>
-            <div class="iui-controls">
-              <a
-                href="#"
-                class="iui-icon"
-              >
-                <!-- Use chevron-left.svg from @itwin/itwinui-icons package. -->
-                <svg
-                  viewBox="0 0 16 16"
-                  class="iui-icon"
-                  aria-hidden="true"
-                >
-                  <path d="m11.3 0 1.4 1.4-6.6 6.6 6.6 6.6-1.4 1.4-8-8z" />
-                </svg>
-              </a>
-              <a class="iui-current">1</a>
-              <a href="#">2</a>
-              <a href="#">3</a>
-              <a href="#">4</a>
-              <a href="#">5</a>
-              <a href="#">6</a>
-              <a href="#">7</a>
-              <span class="iui-ellipsis">&mldr;</span>
-              <a href="#">30</a>
-              <a
-                href="#"
-                class="iui-icon"
-              >
-                <!-- Use chevron-right.svg from @itwin/itwinui-icons package. -->
-                <svg
-                  viewBox="0 0 16 16"
-                  class="iui-icon"
-                  aria-hidden="true"
-                >
-                  <path d="m4.7 0-1.4 1.4 6.6 6.6-6.6 6.6 1.4 1.4 8-8z" />
-                </svg>
-              </a>
-            </div>
-            <div class="iui-location">
-              <label class="iui-input-container">
-                <div class="iui-select">
-                  <div class="iui-select-button">1-10 of 300</div>
-                </div>
-              </label>
+        <div class="iui-right">
+          <label class="iui-input-container">
```

Comment:
Why not a borderless dropdown button?
![image](https://user-images.githubusercontent.com/36186912/121214333-b05da780-c887-11eb-82f8-8c5f4ab7efe0.png)

Reply:
I suppose a borderless dropdown button would work. 🤔

Reply:
Replaced